### PR TITLE
added a bio file for @gvarma

### PR DESCRIPTION
**Pull request by gvarma**
This pull request adds a bio file for @gvarma as requested in #13 
Hey, @gvarma I would :heart: if you had some :time: to put some :eyes: on this and let me know your :thought_balloon: s.

### CODE REVIEW

**Comment by gvarma** [comment]
These changes look good overall, but there are few things we can change.

File: docs/gvarma_file.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Geeta Varma Bio
+
+- Name: Geeta Varma
+- Email: geeta.a.varma@uscis.dhs.com
+- occupation: CM Analyst
+- Location: DC
```
Comment (gvarma) [inline docs/gvarma_file.md]: There are lots of DC people on :earth_americas:

**Comment by tallstuff1964** [comment]
Line 3 needs a correction.

**Comment by tallstuff1964** [comment]
Oops, sorry I reopened the request.

**Comment by gvarma** [comment]
Hi tallstuff1964 I got the message. Testing.

File: docs/gvarma_file.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Geeta Varma Bio
+
+- Name: Geeta Varma
+- Email: geeta.a.varma@uscis.dhs.com
+- occupation: CM Analyst
+- Location: DC 20005
```
Comment (gvarma) [inline docs/gvarma_file.md:6]: Do you think DC zip code is correct?

**Comment by gvarma** [comment]
:+1:  Ship it  (merge it).

**Comment by gvarma** [comment]
Closes #13

**Pull request merged**